python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -380,5 +365,61 @@ def get(self):
             mines = Mine.find_by_name_no_permit(search_term)
         else:
             mines = Mine.find_by_mine_name(name_search)
-        result = list(map(lambda x: {**x.json_by_name(), **x.json_by_location()}, mines))
+        result = list(
```

Comment:
Are we leaving this one cause it's different? wondering why this doesn't have a model and envelopes etc.

Reply:
This one is different as it is returning an object with mixed values from several models. I tried to get it to work with response marshalling but was unable.